Grupo: Patito
Integrantes:
- Jorge 
- Diana

In [2]:
# CARGADO DE LIBRERIAS
import pandas as pd

3. **¿Cuál es la unidad de análisis, de dónde provienen los datos y cuántos registros y variables tiene el conjunto utilizado?** Indiquen qué representa una fila y, si hubo filtros, cuántos registros quedaron después de aplicarlos.

In [6]:
# cargamos datos
seguros = pd.read_csv('data/insurance.csv',sep=',', encoding='iso-8859-1')

print(seguros.shape) #
seguros.head(2)

print("numero de registros: ",seguros.shape[0])
print("numero de columnas o variable: ",seguros.shape[1])

(1338, 7)
numero de registros:  1338
numero de columnas o variable:  7


R. 
- Unidad de análisis: Personas naturales que quieren o estan interesados en adquirir un seguro de salud.
- Origen de datos: Provienen de la plataforma Kaggle, url ...
- Número de Registros: 1338
- Número de columnas: 7

$X_1 = /fraq(5,7)$